lesson_09/oop/oop_app/src/test/java/com/codedifferently/lesson9/mohamedibrahim/PatronTest.java

```diff
@@ -0,0 +1,35 @@
+package com.codedifferently.lesson9.mohamedibrahim;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import org.junit.jupiter.api.Test;
+
+public class PatronTest {
+  @Test
+  void testPatron_create() {
```

Comment:
This test is doing way too much. Based upon the name, I'd expect you act on creating a new `Patron` and to assert that its properties contain the correct values. You should probably split this up into at least three or four individual tests.